test/http_client_test.exs

```diff
@@ -306,6 +306,32 @@ defmodule Closex.HTTPClientTest do
       end
     end
   end
+
+  describe "find_opportunities/1" do
+    test "if some opportunities are found, it returns a list of opportunities" do
+      search_term = ~s(opportunity_status:"S01: Received Offer Request")
+
+      use_cassette "find_opportunities" do
+        {:ok, result} = find_opportunities(search_term)
+        assert %{"data" => opportunities, "total_results" => 48} = result
+        assert is_list(opportunities)
+        assert Enum.count(opportunities) == 48
+        opportunity = hd(opportunities)
+        assert opportunity["status_label"] == "S01: Received Offer Request"
+      end
+    end
+    test "if no opportunities are found, it returns an empty list" do
```

Comment:
New line between tests please

Reply:
This entire file is formatted poorly. I was intending to wait until elixir 1.6 is released then run the autoformatter.